share/commands/agent-run

```diff
@@ -83,4 +90,8 @@ if [ -e /opt/rudder/etc/disable-agent ]; then
   exit 1
 fi
 
+if [ "${UPDATE}" = "true" ]; then
```

Comment:
You should reuse the provided options, otherwise you would break user expectations.